scm/pr.go

```diff
@@ -48,6 +48,7 @@ type (
 		Size   int
 		Open   bool
 		Closed bool
+		Merged bool
```

Comment:
this is added in a common file, we would need addition for all providers for this if they support

Reply:
Gitlab now has 4 states -> `opened, closed, locked, merged`
Bitbucket also now has 4 -> `OPEN, MERGED, DECLINED, SUPERSEDED`
Till now everywhere we have this condition though
`if opts.Open && opts.Closed {
	params.Set("state", "all")
}`

Reply:
this might affect the usage of this method

Reply:
one requirement to merging new functionality in this library is that it needs to be implemented for 3 or more providers to ensure the abstractions are correct.  The differences you highlighted are the reason this is required.

Reply:
removed